src/student_success_tool/features/pdp/student.py

```diff
@@ -62,6 +62,11 @@ def add_features(
             )
             for yr in credits_years
         },
+        # not sure if we want to just re-code the original column here or make a new one and call it something else, ex. pell_grant_recipient
```

Comment:
how about we call it `student_is_pell_recipient_first_year`, for consistency with the other student-level features? and to follow the existing pattern, let's rename the `change_pell_to_T_F` function to the same thing.